console/src/app/modules/providers/provider-oauth/provider-oauth.component.html

```diff
@@ -0,0 +1,111 @@
+<cnsl-create-layout
+  title="{{ id ? ('IDP.DETAIL.TITLE' | translate) : ('IDP.CREATE.TITLE' | translate) }}"
+  (closed)="close()"
+>
+  <div class="oauth-create-content">
+    <div class="title-row">
+      <img class="idp-logo" src="../../../assets/images/idp/oauth.svg" alt="oauth" />
+
+      <h1>{{ 'IDP.CREATE.OAUTH.TITLE' | translate }}</h1>
+      <mat-spinner diameter="25" *ngIf="loading" color="primary"></mat-spinner>
+    </div>
+
+    <p class="desc cnsl-secondary-text">{{ 'IDP.CREATE.OAUTH.DESCRIPTION' | translate }}</p>
+
+    <form [formGroup]="form" (ngSubmit)="submitForm()">
+      <cnsl-form-field class="formfield">
+        <cnsl-label>{{ 'IDP.NAME' | translate }}</cnsl-label>
+        <input cnslInput formControlName="name" />
+      </cnsl-form-field>
+
+      <cnsl-form-field class="formfield">
+        <cnsl-label>{{ 'IDP.AUTHORIZATIONENDPOINT' | translate }}</cnsl-label>
+        <input cnslInput formControlName="authorizationEndpoint" />
+      </cnsl-form-field>
+
+      <cnsl-form-field class="formfield">
+        <cnsl-label>{{ 'IDP.TOKENENDPOINT' | translate }}</cnsl-label>
+        <input cnslInput formControlName="tokenEndpoint" />
+      </cnsl-form-field>
+
+      <cnsl-form-field class="formfield">
+        <cnsl-label>{{ 'IDP.USERENDPOINT' | translate }}</cnsl-label>
+        <input cnslInput formControlName="userEndpoint" />
+      </cnsl-form-field>
+
+      <cnsl-form-field class="formfield">
+        <cnsl-label>{{ 'IDP.IDATTRIBUTE' | translate }}</cnsl-label>
+        <input cnslInput formControlName="idAttribute" />
+      </cnsl-form-field>
```

Comment:
Do we validate these fields?

Reply:
backendwise they are not validated so it does not make sense to validate in frontend either